pkg/externalresource/resourcemeta/model.go

```diff
@@ -46,6 +46,30 @@ func (m *ResourceMeta) ToQueryResourceResponse() *pb.QueryResourceResponse {
 	}
 }
 
+// Columns is used for updating the orm model
+func (m *ResourceMeta) Columns() []string {
```

Comment:
Good suggestion. 
It's used in `Updatexxx` interface

Reply:
It is related to some gorm inner implement. 
I add a TODO here and do it later.
In this pr, I will move it to a variable to avoid useless object allocate.